jira7/Package.md

```diff
@@ -1,72 +1,94 @@
+and update issues, work with JIRA user accounts, etc. It handles basic authentication and provides 
+auto completion and type conversions.
+
+**JIRA Project Operations**
+
+The `wso2/jira` package contains operations to create new JIRA projects, update or delete existing projects, and get all 
+the information using either the ID or key of the project. It also contains operations for adding or removing users and 
+groups related to a project role, viewing user account details of the project lead, viewing assignable issue types for a 
+given project, etc.
 
-| Ballerina Version | Jira REST API Version |
-|:-------------------:|:-------------------:|
-|0.970.0-beta14|7.2.2|
+**JIRA Project Category Operations**
 
-## Getting started
+The `wso2/jira` package contains operations that get all available project categories, delete existing categories, and 
+create new project categories.
 
-1. Refer `https://ballerina.io/learn/getting-started/` to download Ballerina and install tools.
+**JIRA Project Component Operations**
 
-2. Ballerina Jira connector currently provides basic authentication as the authentication method. Therefore obtain your 
-   Jira user account credentials(username and password).If you currently dont have a Jira account, you can create a new Jira account from 
-   [JIRA Sign-Up Page](https://id.atlassian.com/signup?application=mac&tenant=&continue=https%3A%2F%2Fmy.atlassian.com).
-   
-   *Note -JIRA’s REST API is protected by the same restrictions which are provided via JIRAs standard web interface.
-    This means that if you do not have valid jira credentials, you are accessing JIRA anonymously. Furthermore, 
-    if you log in and do not have permission to view something in JIRA, you will not be able to view it using the 
-    Ballerina JIRA Connector as well.*
+The `wso2/jira` package contains operations that get all details of a given project component, delete existing 
+components, create a new project component related to a specific project, etc.
 
-3. Create a new Ballerina project by executing the following command.
-      ```shell
-        <PROJECT_ROOT_DIRECTORY>$ ballerina init`
-      ```
-  
-4. Import the jira package to your Ballerina program as follows.This will download the jira7 artifacts from the 
-`ballerina central` to your local repository.
+**JIRA Issue Operations**
+
+The `wso2/jira` package contains operations that get all the details of a given issue using the issue key, delete existing 
+issues, create new issues, etc.
+
+## Compatibility
+|                    |    Version     |  
+| :-----------------:|:--------------:| 
+| Ballerina Language | 0.970.0-beta15 |
+| JIRA REST API      |    7.2.2       |  
 
-    ```ballerina
-       import wso2/jira7;
-    ```
+## Sample
+The JIRA connector can be used to get summaries of all projects, create a project, and create an issue. First, import 
```

Comment:
Delete "The JIRA connector can be used to get summaries of all projects, create a project, and create an issue."

Reply:
Fixed.